nss-day-cohort-45/holiday-road-frozen-assets
# Caa save itinerary

### Changes made:
1. Added some styling to aside section (still working on it)
2. Got the itineraries to show up in DOM (still working on save feature)
3. Button now disabled by default (no logic added yet to enable it)

How to check: 
1. fetch branch and serve
2. check that dummy itinerary entries show up in aside
3. make sure my styling didn't break the appearance of the previews section
4. Check that button is disabled
